deco-sites/otica-isabela
# remove test justify from advantages section within pdp

## What is this contribution about?

Removing text-justify from Advantages Section within PDP.

## How to test it?

Go to any PDP and check the advantages Section. The text that appear when hovering it's not justified anymore.
